index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Abdessittir Harkati</title>
+</head>
+<body>
+    <main class="main">
+        <header class="header">
+            <h2>Welcome</h2>
+            <button>
+                <img class="menu-icon" src="./media/Union.png" alt="menu" />
+            </button>
```

Comment:
- [ ] Great job with your project 👍 . Kindly wrap your `hamburger menu` in a `nav` tag 🙏 . The `nav` tag is for semantic reasons.